docs/001_develop/03_client-capabilities/008_interaction/date-picker.mdx

```diff
@@ -0,0 +1,283 @@
+            </rapid-date-picker>
+          );
+        }
+        ```
+    </TabItem>
+    <TabItem value="angular">
+      Usage
+      ```typescript
+      import { AfterViewInit, Component, CUSTOM_ELEMENTS_SCHEMA, ElementRef, ViewChild } from '@angular/core';
+
+      @Component({
+      selector: 'date-picker-example',
+      template: `
+        <rapid-date-picker (value-changed)="handleValueChanged($event)">
+        </rapid-date-picker>
+        `,
+        standalone: true,
+        schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      })
+      export class DatePickerExampleComponent implements AfterViewInit {
+          valueChanged(e) {
+            console.log(e.detail);
+          }
+      }
+      ```
+    </TabItem>
+</Tabs>
+
+## DOM API
+
+The sections below provide property and attribute binding examples for Genesis Component syntax. The HTML closing tag is not included.
+
+### Attributes
+
+<table>
+    <thead>
+        <tr>
+            <th>Attribute</th>
+            <th>Type</th>
+            <th>Use</th>
+            <th>Example</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>format</td>
+            <td><code>string</code></td>
+            <td>The date format shown defaults to <code>MM-DD-YYYY</code> and can accept any <a href="https://day.js.org/docs/en/display/format">valid dayjs format</a></td>
+            <td>
+
+            ```typescript
+            <rapid-date-picker format="DD-MM-YYYY">
+            ```
+
+            </td>
+        </tr>
+        <tr>
+            <td>lang</td>
+            <td><code>string</code></td>
+            <td>Determines the defuault locale. Defaults to <code>en-US</code> and can accept any <a href="https://day.js.org/docs/en/i18n/changing-locale">valid day js lang</a></td>
+            <td>
+
+            ```typescript
+            <rapid-date-picker lang="en-US">
+            ```
+
+            </td>
+        </tr>
+        <tr>
+            <td>visible</td>
+            <td><code>boolean</code></td>
+            <td>Displays the calendar by default. Defaults to <code>false</code></td>
+            <td>
+
+            ```typescript
+            <rapid-date-picker visible>
+            ```
+
+            </td>
+        </tr>
+        <tr>
+            <td>inline-calendar</td>
+            <td><code>boolean</code></td>
+            <td>Displays the calendar as an inline block element as opposed to being absolutely positioned relative to it's parent. Defaults to <code>false</code></td>
+            <td>
+
+            ```typescript
+            <rapid-date-picker inline-calendar>
+            ```
+
+            </td>
+        </tr>
+        <tr>
+          <td>hide-weekends</td>
+          <td><code>boolean</code></td>
+          <td>Hides Saturday and Sundays restricting user to selecting only week days. Defaults to <code>false</code></td>
+          <td>
+            ```typescript
+            <rapid-date-picker hide-weekends>
+            ```
```

Comment:
With these ones further down the table you need the left align as well as space above and below the code fense